WcaOnRails/app/models/user.rb

```diff
@@ -220,6 +220,8 @@ def locked_account?
   scope :delegates, -> { where.not(delegate_status: nil) }
   scope :senior_delegates, -> { where(delegate_status: "senior_delegate") }
 
+  scope :african_delegates, -> { where(senior_delegate_id: User.where("region like ?", "%Africa%").map(&:id)).or(User.where("region like ?", "%Africa%")) }
```

Comment:
I wonder if we could change this to a scope like `african_senior_delegate` and then we could change the caller to do something like:

```
User.african_senior_delegate.subordinate_delegates + User.african_senior_delegate
```

I think this would allow simplifying the scope a little bit to something like:

```
scope :african_senior_delegate, -> { senior_delegate.where("region like ?", "%Africa%") }
```

Reply:
Nevermind, see my other comment above.